apps/dash-manufacture-spc-dashboard/app.py

```diff
@@ -0,0 +1,1209 @@
+                            ),
+                            value=params[1],
+                        ),
+                    ],
+                ),
+                html.Div(
+                    id="value-setter-menu",
+                    # className='six columns',
+                    children=[
+                        html.Div(id="value-setter-panel"),
+                        html.Br(),
+                        html.Div(
+                            id="button-div",
+                            children=[
+                                html.Button("Update", id="value-setter-set-btn"),
+                                html.Button(
+                                    "View current setup",
+                                    id="value-setter-view-btn",
+                                    n_clicks=0,
+                                ),
+                            ],
+                        ),
+                        html.Div(
+                            id="value-setter-view-output", className="output-datatable"
+                        ),
+                    ],
+                ),
+            ],
+        ),
+    ]
+
+
+ud_usl_input = daq.NumericInput(
+    id="ud_usl_input", className="setting-input", size=200, max=9999999
+)
+ud_lsl_input = daq.NumericInput(
+    id="ud_lsl_input", className="setting-input", size=200, max=9999999
+)
+ud_ucl_input = daq.NumericInput(
+    id="ud_ucl_input", className="setting-input", size=200, max=9999999
+)
+ud_lcl_input = daq.NumericInput(
+    id="ud_lcl_input", className="setting-input", size=200, max=9999999
+)
+
+
+def build_value_setter_line(line_num, label, value, col3):
+    return html.Div(
+        id=line_num,
+        children=[
+            html.Label(label, className="four columns"),
+            html.Label(value, className="four columns"),
+            html.Div(col3, className="four columns"),
+        ],
+        className="row",
+    )
+
+
+def generate_modal():
+    return html.Div(
+        id="markdown",
+        className="modal",
+        children=(
+            html.Div(
+                id="markdown-container",
+                className="markdown-container",
+                children=[
+                    html.Div(
+                        className="close-container",
+                        children=html.Button(
+                            "Close",
+                            id="markdown_close",
+                            n_clicks=0,
+                            className="closeButton",
+                        ),
+                    ),
+                    html.Div(
+                        className="markdown-text",
+                        children=dcc.Markdown(
+                            children=(
+                                """
+                        ###### What is this mock app about?
+
+                        'dash-manufacture-spc-dashboard` is a dashboard for monitoring read-time process quality along manufacture production line. 
```

Comment:
same thing here, one is a single apostrophe and one is a backtick

Reply:
💯